README.md

```diff
@@ -46,25 +48,66 @@ title: document number 2
 
 For more Markdown features, including autogenerated table of contents, refer to [Docusaurus docs](https://docusaurus.io/docs/en/doc-markdown.html).
 
-### Adding images
+### Tips for documenting
 
-Save the image in website/static/assets and reference it like this:
+- To add images, save the image in website/static/assets and reference it like this:
 
 ```md
 ![](/assets/admin-dashboard.png "Reaction Dashboard")
 ```
 
-### Cross Linking
-
-Reference other articles by their filename:
+- To link to other articles, reference other articles by their filename:
 
 ```md
 Refer to the [FAQs](faqs.md) article
 ```
 
-### Syntax highlighting
+- To enable code syntax highlighting, add `js`, `jsx`, `graphql`, `html`, `sh`, `git`, `yaml` and [more](https://github.com/reactioncommerce/reaction-docs-static/issues/47) after the ```.
+
+### Sign your commits
```

Comment:
I'd prefer to keep our language about the DCO consistent between repos. What do you think about using the same content as in the `reaction` project?
https://github.com/reactioncommerce/reaction/blob/master/README.md#developer-certificate-of-origin

Reply:
Not sure why this is showing the Syntax Highlighting section, but it's about the `Sign your commits` section.

Reply:
ok! i got this from the component library one. the language has shifted! ill switch it to the reaction one.